pkgs/applications/misc/furtherance/default.nix

```
@@ -0,0 +1,55 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform, appstream-glib, desktop-file-utils
+, glib, libadwaita, meson, ninja, pkg-config, python3, wrapGAppsHook4, dbus
+, gtk4, sqlite }:
+stdenv.mkDerivation rec {
+  pname = "Furtherance";
```

We use lowercase `pname`s.

```suggestion
  pname = "furtherance";
```